pipeline/ingest/ephys.py

```diff
@@ -912,21 +912,23 @@ def _get_sess_dir(rigpath, h2o, sess_datetime):
         dglob = '{}_*_imec[0-9]'.format(
             sess_datetime.date().strftime('%Y%m%d')) + '/{}'
     else:
-        sess_dirs = list((rigpath / h2o).glob('*{}_{}_*'.format(
-            h2o, sess_datetime.date().strftime('%m%d%y'))))
-        for sess_dir in sess_dirs:
-            try:
-                npx_meta = NeuropixelsMeta(next(sess_dir.rglob('{}_*.ap.meta'.format(h2o))))
-            except StopIteration:
-                continue
-            # ensuring time difference between behavior-start and ephys-start is no more than 2 minutes - this is to handle multiple sessions in a day
-            start_time_difference = abs((npx_meta.recording_time - sess_datetime).total_seconds())
-            if start_time_difference <= 120:
-                dpath = sess_dir
-                dglob = '{}_{}_*_imec[0-9]'.format(h2o, sess_datetime.date().strftime('%m%d%y')) + '/{}'  # probe directory pattern
-                break
-            else:
-                log.info('Found {} - difference in behavior and ephys start-time: {} seconds (more than 2 minutes). Skipping...'.format(sess_dir, start_time_difference))
+        date_strings = [sess_datetime.date().strftime('%m%d%y'), sess_datetime.date().strftime('%Y%m%d')]
+        for date_string in date_strings:
+            sess_dirs = list(pathlib.Path(rigpath, h2o).glob('*{}*{}_*'.format(h2o, date_string)))
```

Comment:
Instead of having to create another loop for date_strings, perhaps something like this:

```suggestion
        sess_dirs = (list(pathlib.Path(rigpath, h2o).glob('*{}*{}_*'.format(h2o, sess_datetime.date().strftime('%m%d%y')))) +
                            list(pathlib.Path(rigpath, h2o).glob('*{}*{}_*'.format(h2o, sess_datetime.date().strftime('%Y%m%d')))))
```

Reply:
ah yes, you're correct, yeah I think the loop would be cleaner, more readable